docs/rule/android_library.soy

```diff
@@ -157,6 +157,15 @@ compiled class files and resources.
   {/param}
 {/call}
 
+{call buck.arg}
+  {param name: 'provided_deps' /}
+  {param default : '[]' /}
+  {param desc}
+  This is the same as in <a href="{ROOT}rule/java_library.html">
+  <code>java_library</code></a>.
+  {/param}
+{/call}
```

Comment:
It would be best here to do a `{call jvm_common.provided_deps}` here, like we do in `java_library.soy`.  I suspect, but haven't verified, that we want this to apply for `android_binary` and not `java_binary`, which is what `java_library` says.

Reply:
Yes, thanks for the catch! This should be `android_binary` as an `android_library` cannot be packaged into a `java_library`. Updated, along with a similar change for `exported_deps`.